docs/install/config-ha.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Configuring high-availability components"
+linkTitle: "Configuring high-availability components"
+weight: 50
+type: "docs"
+---
+
+Active/passive high availability (HA) is a standard feature of Kubernetes APIs that helps to ensure that APIs stay operational if a disruption occurs. In an HA deployment, if an active controller crashes or is deleted, another controller is available to take over processing of the APIs that were being serviced by the controller that is now unavailable.
+
+Active/passive HA in Knative is available through leader election, which is enabled by default after the Knative Serving control plane is installed.
+
+When using a leader election HA pattern, instances of controllers are already scheduled and running inside the cluster before they are required. These controller instances compete to use a shared resource, known as the leader election lock. The instance of the controller that has access to the leader election lock resource at any given time is referred to as the leader.
+
+HA functionality is available on Knative for the `autoscaler-hpa`, `controller`, `activator` , `kourier-control`, `kourier-gateway`, `istio-pilot`, and `istio-gateway` components.
```

Comment:
I'm not sure if pilot and gateway are the right level of info to expose... Maybe 'and the controllers integrating Knative with Kourier and Istio ingresses' is better? I don't think many users will be understand exactly what these names mean. WDYT?

Reply:
In general, as I proposed above, I'd just keep knative components.

Reply:
Cool, will do. Thanks folks.

Reply:
`istio-pilot` and `istio-gateway` are not provided and controlled by Knative. They are provided by Istio. So better to exclude them.

If we want to include kourier-control, do we also want to include istio-controller and contour-controller?